src/XRP/xrp-client-decorator.ts

```diff
@@ -2,7 +2,7 @@ import { Wallet } from 'xpring-common-js'
 import { BigInteger } from 'big-integer'
 import TransactionStatus from './transaction-status'
 import RawTransactionStatus from './raw-transaction-status'
-import XRPTransaction from './model/xrp-transaction'
+import { XRPTransaction } from './model/xrp-transaction'
```

Comment:
Are we just going to wait until deleting the old code to replace the references to the old class names?

Reply:
See #539. 

I think I have to deprecate / migrate some more XRP classes than initially intended.